### ISSUE TITLE
Text/background contrast of links fails W3C color contrast check

### ISSUE THREAD

**Issue by ebuie** [tjmonsi/chi-interactive-program-v2#22]
The green-on-white text of links to show and hide abstracts (and probably also the white-on-green of the buttons to search and filter) fails to meet the color contrast requirements for accessibility as specified in the WCAG. It's probably too late for this to be rectified for this year, the conference being past, but we should pay attention to it for future conference websites. 
The "toggle high contrast" button is not a solution. Most people just want adequate contrast in the "regular" version.

This image is from the WebAIM contrast checker, at https://webaim.org/resources/contrastchecker/
<img width="723" alt="screen shot 2018-05-04 at 10 31 28" src="https://user-images.githubusercontent.com/16671745/39621560-1c7ceb38-4f87-11e8-97a5-221dd3256062.png">
